test/Button.swift

```diff
@@ -68,6 +68,16 @@ final class Button: UIButton {
         setTitle(title, for: .normal)
     }
 
+    override init() {
+        super.init(frame: .zero)
+
+        setTitleColor(.black, for: .normal)
+        setTitle("Title", for: .normal)
+
+        setBackgroundColor(color: .primary, forState: .normal)
+        setBackgroundColor(color: .primary.withAlphaComponent(0.8), forState: .highlighted)
+    }
+
```

Comment:
This initializer is not acceptable by the compiler since it's a designated initializer. Changing it as `required init` and moving it to top of the `override init(frame: NSRect)` method would be better.